install_config/install/prerequisites.adoc

```diff
@@ -89,9 +89,9 @@ endif::[]
 * Minimum 15 GB hard disk space for the file system containing *_/var/_*. image:redcircle-1.png[]
 * Minimum 1 GB hard disk space for the file system containing *_/usr/local/bin/_*.
 * Minimum 1 GB hard disk space for the file system containing the system's temporary directory. image:redcircle-2.png[]
-* An additional minimum 15 GB unallocated space to be used for Docker's storage
+* An additional minimum 15 GB unallocated space (per system running containers) to be used for Docker's storage
```

Comment:
s/(per system running containers) to be used for/per system that runs containers for